eslint/babel-eslint-config-internal/package.json

```diff
@@ -1,21 +1,15 @@
 {
-  "name": "eslint-config-babel",
-  "version": "9.0.0",
-  "description": "ESLint config for babel",
+  "name": "@babel/eslint-config-internal",
+  "version": "7.7.0",
```

Comment:
Does this seem right? Or should I leave this as is for now until we actually publish this?

Reply:
As long as it's <= than the current Babel version, it should be ok. Mabe `0.0.0` so that it's clear that it isn't published yet?